portfolio/src/main/webapp/js/languages.js

```diff
@@ -0,0 +1,187 @@
+/**
+ * A list of languages and ISO 639 codes, scraped automatically from https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes.
```

Comment:
Does the translation API handle all of these languages?

Reply:
Actually I'm not sure, patched this with new list scraped directly from cloud website.